mobile-app/app/screens/AppNavigator/screens/Auctions/components/BidHistory.tsx

```diff
@@ -0,0 +1,205 @@
+
+interface BidHistoryProps {
+  vaultId: string
+  liquidationHeight: number
+  batchIndex: number
+  loanDisplaySymbol: string
+  loanSymbol: string
+  minNextBidInToken: string
+}
+
+export function BidHistory (props: BidHistoryProps): JSX.Element {
+  const client = useWhaleApiClient()
+  const dispatch = useDispatch()
+  const blockCount = useSelector((state: RootState) => state.block.count) ?? 0
+  const bidHistory = useSelector((state: RootState) => state.auctions.bidHistory)
+  const { getTokenPrice } = useTokenPrice()
+  const isFocused = useIsFocused()
+
+  useEffect(() => {
+    if (isFocused) {
+      dispatch(fetchBidHistory({
+        vaultId: props.vaultId,
+        liquidationHeight: props.liquidationHeight,
+        batchIndex: props.batchIndex,
+        client: client,
+        size: 200
+      }))
+    } else {
+      dispatch(auctions.actions.resetBidHistory())
+    }
+  }, [blockCount, isFocused])
+
+  return (
+    <ThemedFlatList
+      data={bidHistory}
+      renderItem={({ item, index }: { item: VaultAuctionBatchHistory, index: number }): JSX.Element => {
+        return (
+          <BidHistoryItem
+            bidIndex={bidHistory.length - index}
+            bidAmount={item.amount}
+            loanDisplaySymbol={props.loanDisplaySymbol}
+            bidderAddress={item.from}
+            bidAmountInUSD={getTokenPrice(props.loanSymbol, new BigNumber(item.amount))}
+            isLatestBid={index === 0}
+            bidBlockTime={item.block.time}
+          />
+        )
+      }}
+      ListEmptyComponent={() => (
+        <EmptyBidHistory
+          minNextBid={props.minNextBidInToken}
+          displaySymbol={props.loanDisplaySymbol}
+        />
+      )}
+      keyExtractor={(item: VaultAuctionBatchHistory) => item.id}
+      contentContainerStyle={[tailwind('p-4 pb-8'), tailwind({ 'pb-44': Platform.OS !== 'android' && Platform.OS !== 'ios' })]}
+      light={tailwind('bg-gray-50')}
+      style={tailwind('-mb-1')}
+      initialNumToRender={5}
+      windowSize={2}
+    />
+  )
+}
+
+interface BidHistoryItemProps {
+  bidIndex: number
+  bidAmount: string
+  loanDisplaySymbol: string
+  bidderAddress: string
+  bidAmountInUSD: BigNumber
+  isLatestBid: boolean
+  bidBlockTime: number
+}
+
+function BidHistoryItem (props: BidHistoryItemProps): JSX.Element {
+  const bidTime = useBidTimeAgo(props.bidBlockTime)
+  return (
+    <ThemedView
+      light={tailwind({ 'bg-white border-gray-100': props.isLatestBid, 'bg-gray-100 border-gray-100': !props.isLatestBid })}
+      dark={tailwind({ 'bg-gray-800 border-gray-900': props.isLatestBid, 'bg-gray-900 border-gray-800': !props.isLatestBid })}
+      style={tailwind('border rounded px-4 py-3 mb-1')}
+      testID={`bid_${props.bidIndex.toString()}`}
+    >
+      <View style={tailwind('flex flex-row justify-between mb-2 items-center')}>
+        <ThemedView
+          style={tailwind('px-1 rounded-sm')}
+          light={tailwind('bg-blue-500 text-white')}
```

Comment:
Should this be only blue when it's the latest build? Based from the latest designs

![image](https://user-images.githubusercontent.com/25013382/156486346-953ebd0b-8c49-41ab-9c5c-b6a09282e8ad.png)